### PR TITLE
feat: add admin cuisine controller rspec

### PR DESCRIPTION
**Pull request by kien2572001**
## Related Tickets
- ticket redmine

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by Huy0110** [comment]
ok

**Comment by kien2572001** [comment]
ready

**Comment by kien2572001** [comment]
ready

**Comment by kien2572001** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -75,6 +75,8 @@ gem "will_paginate", "~> 4.0"
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html
   gem "debug", platforms: %i(mri mingw x64_mingw)
+  gem "factory_bot_rails"
+  gem "rails-controller-testing", "~> 1.0"
```
Comment (vanvtt-0952) [inline Gemfile]: cái này bỏ trong group test thôi chứ em

File: spec/controllers/admin/cuisines_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,178 @@
+require "rails_helper"
+
+RSpec.describe Admin::CuisinesController, type: :controller do
+  let(:admin_user) { FactoryBot.create(:admin_user) }
+  #sign_in_as :admin_user
+  
+
+  describe "GET #index" do
```
Comment (vanvtt-0952) [inline spec/controllers/admin/cuisines_controller_spec.rb]: thế nếu chưa login thì sao
login nhưng k phải role admin thì sao 
chauw thấy handle nè

**Comment by kien2572001** [comment]
ready

**Comment by kien2572001** [comment]
ready

**Pull request merged**